### ISSUE TITLE
你好，我是用2d图像训练时遇到问题

### ISSUE THREAD

**Issue by NBIronman** [MontaEllis/Pytorch-Medical-Segmentation#46]
在使用您的代码时，我设置2d图像训练，在main.py文件中255行和277行，应该怎么修改吗？

**Comment by NickWang1122** [comment]
你好,我也遇到了这个问题,
model_output_one_hot = torch.nn.functional.one_hot(labels, num_classes=hp.out_class+1).permute(0,4,1,2,3)是这行代码报错
请问解决了吗？

**Comment by MontaEllis** [comment]
抱歉，我会在最近更新一下代码

**Comment by pipixiaipip** [comment]
> 在使用您的代码时，我设置2d图像训练，在main.py文件中255行和277行，应该怎么修改吗？

你好，请问您解决了吗？该如何解决啊
